src/en/clean-copy/04-Part III. The New Hope/30.md

```diff
@@ -1,56 +1,56 @@
 ### USDA 56013
 
-Until some moment, the craft revolution, though opposed to mass production, still hadn't made anything truly revolutionary: those English-style ales and porters by Anchor, New Albion, Boulder, Newman, and DeBakker were representing the classical styles, maybe with just some peculiarities. At the beginning of the 80s, with the third wave of craft startups coming, the situation began to change.
```

Comment:
How about:

By 1980, the scales had tipped. Though opposed to mass production, the craft revolution still had not made anything....

Reply:
@JoRembert  Maybe just ‘By 1980, though opposed to mass production, the craft revolution still had not made anything'?